tools/compare-wp-performance/run.sh

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env bash
```

Comment:
Nit-pick, but it would be great if you could make this script work regardless from where you run it. Right now it only works if you first `cd` into the `tools/compare-wp-performance` directory. Running `./tools/compare-wp-performance/run.sh` from the root directory gives errors due to the locations.